freak/macros.xml

```diff
@@ -3,7 +3,6 @@
 <macros>
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="@VERSION@">freak</requirement>
```

Comment:
Need to keep this requirement if the freak tool and post-processing tool are to share macros